marklogic-data-hub/src/server-side/impl/trace-lib.xqy

```diff
@@ -302,7 +302,7 @@ declare %private function trace:write-error-trace(
             "/" || $trace/*:trace/*:traceId || $extension,
             $trace,
             xdmp:default-permissions(),
-            ("trace", $trace/*:trace/*:type)
+            ("trace", $trace/*:trace/*:flow-type, $trace/*:trace/*:flowType)
```

Comment:
This looks like something we missed during a name change or format introduction.  time to introduce some ml-unit-test?